pkg/system/phase4_configuring.go

```diff
@@ -270,6 +271,30 @@ func (r *Reconciler) SetDesiredDeploymentEndpoint() error {
 	podSpec.SecurityContext.RunAsGroup = &rootUIDGid
 	podSpec.ServiceAccountName = "noobaa-endpoint"
 
+	honor := corev1.NodeInclusionPolicyHonor
+	disableDefaultTopologyConstraints, found := r.NooBaa.ObjectMeta.Annotations[nbv1.SkipTopologyConstraints]
+	if podSpec.TopologySpreadConstraints != nil {
+		r.Logger.Infof("deployment %s TopologySpreadConstraints already exists, leaving as is", r.DeploymentEndpoint.Name)
```

Comment:
this might be too verbose, since it is expected to be the case for most reconciles.

Reply:
Hi @dannyzaken, Guy sugested this changes  : https://github.com/noobaa/noobaa-operator/pull/1138#discussion_r1257818370